ksqldb-cli/src/test/java/io/confluent/ksql/cli/console/UnclosedQuoteCheckerTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.cli.console;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class UnclosedQuoteCheckerTest {
```

Comment:
Can you add some test cases please?

* where `--` appears _within a quoted string
* where `--` appears _after_ a quoted string is closed?
* where a quoted string contains a single quote, e.g. 'this string''contains a single quote'
* some with double quotes in quoted strings just for the hell of it.